test/distributed/org/apache/cassandra/distributed/Constants.java

```diff
@@ -42,4 +42,12 @@ public final class Constants
     public static final String KEY_DTEST_FULL_STARTUP = "dtest.api.startup.full_startup";
 
     public static final String KEY_DTEST_JOIN_RING = "dtest.api.startup.join_ring";
+
+    /**
+     * Adds a timeout to instance startup so the tests don't block forever when startup bugs can happen.  This should
+     * almost always be paired with {@link #KEY_DTEST_API_STARTUP_FAILURE_AS_SHUTDOWN} as the startup still needs to cleanup.
+     *
+     * {@code c.set(Constants.KEY_DTEST_API_STARTUP_FAILURE_AS_SHUTDOWN, false); }
+     */
+    public static final String KEY_DTEST_STARTUP_TIMEOUT = "dtest.api.startup.timeout";
```

Comment:
> 9) JVM Dtest now supports startup timeouts, this is to avoid issues where startup will take infinity (due to bugs) causing CI to throw away the logs.

when accord hits a bug bootstrap is blocked forever, which then means we get 0 logs in CI as we just get "timeout"... we *need* to bound our startup logic in order to get CI to tell us we are broken and why